packages/wallet/test/hardware/ledger/LedgerKeyAgent.test.ts

```diff
@@ -298,6 +314,317 @@ describe('LedgerKeyAgent', () => {
         } = await wallet.finalizeTx({ tx: unsignedTx });
         expect(signatures.size).toBe(2);
       });
+
+      describe('conway-era', () => {
+        describe('ordinary tx mode', () => {
+          it('can sign a transaction with Registration certs', async () => {
+            const txBuilder = wallet.createTxBuilder();
+            txBuilder.partialTxBody.certificates = [
+              {
+                __typename: Cardano.CertificateType.Registration,
+                deposit: 5n,
+                stakeCredential: getStakeCredential(
+                  (await firstValueFrom(wallet.delegation.rewardAccounts$))?.[0].address
+                )
+              }
+            ];
+            const tx = txBuilder
+              .addOutput(
+                txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+              )
+              .build();
+
+            expect(await tx.sign()).toBeTruthy();
```

Comment:
In other tests, we are not signing tx directly but `wallet.finalizeTx()` to ensure the whole process is good. Maybe it would be good to follow that ha? What do you think?

Reply:
but `finalizeTx` is marked as deprecated?

Reply:
hmmm, then we should change other test cases ( in Trezor as well ). Not necessarily in the scope of this PR.
cc @AngelCastilloB @mkazlauskas ?

Reply:
I think it's fine to use either for now, refactor can take place if/when `finalizeTx` is removed. Up to you guys :)

Reply:
Amazing, thanks @mkazlauskas <3 
@vetalcore I'm resolving this conv. and once we have more time we can do a refactor in both HW packages.